packages/elements-core/src/components/ResponseExamples/ResponseExamples.tsx

```diff
@@ -40,18 +40,34 @@ export const ResponseExamples = ({ httpOperation, responseMediaType, responseSta
     />
   );
 
-  return (
-    <Panel rounded isCollapsible={false}>
-      <Panel.Titlebar>{examplesSelect || <Text color="body">Response Example</Text>}</Panel.Titlebar>
-      <Panel.Content p={0}>
+  const ResponseExamplesViewer: React.FC<{ example: string }> = ({ example }) => {
+    const [show, setShow] = React.useState<boolean>(false);
+    const exceededSize = example.split(/\r\n|\r|\n/).length > 500;
+
+    if ((exceededSize && show) || !exceededSize)
+      return (
         <CodeViewer
-          aria-label={responseExample}
+          aria-label={example}
           noCopyButton
           maxHeight="400px"
           language="json"
-          value={responseExample}
+          value={example}
           showLineNumbers
         />
+      );
+
+    return (
+      <Box p="auto" onClick={() => setShow(true)}>
```

Comment:
If it's clickable, we should make some kind of button out of it, also for accessibility reasons.